modules/tensor_mechanics/test/tests/ad_anisotropic_plasticity/gold/ad_aniso_plasticity_x_out.csv

```diff
@@ -1,27 +1,27 @@
-3.5257789253555e-05,1.5513282159785e-07,2.0147307988292e-08,1.7976931348623e+308,1.9721545592816e-07,1.9862111312277e-09,0.00047010385016905,0,0.0014103115500434
-3.5420678716233e-05,1.6288946267774e-07,2.0230058258356e-08,0.0007700847028815,2.0216364612429e-07,2.0986727005194e-09,0.00047227572129575,4.2275117632129e-10,0.001416827155644
-3.5591712652044e-05,1.7103393581163e-07,2.0161804755584e-08,0.00018138924912393,2.199406542118e-07,2.5392270045777e-09,0.00047455617477488,2.3070836068733e-09,0.001423668502841
-3.5771298284646e-05,1.7958563260221e-07,2.0247328613533e-08,0.00018063170898845,2.4022624234592e-07,3.0526063517519e-09,0.00047695064194951,4.2939121796223e-09,0.0014308519091755
-3.5959863198879e-05,1.8856491423232e-07,2.033637319363e-08,0.00017985087161049,2.6160717342515e-07,3.5941950299877e-09,0.00047946484037058,6.3891192703349e-09,0.0014383945173133
-3.6157856358823e-05,1.9799315994394e-07,2.0429050470759e-08,0.0001790468763399,2.841456031654e-07,4.16564004898e-09,0.00048210475368878,8.5989425749449e-09,0.0014463142537988
-3.6365749176764e-05,2.0789281794114e-07,2.0525472677013e-08,0.00017821960400193,3.0790742797705e-07,4.7686797446271e-09,0.00048487664989703,1.0930001421141e-08,0.0014546299656564
-3.6584036635602e-05,2.1828745883819e-07,2.0625752670842e-08,0.00017736904399161,3.3296252039182e-07,5.4051770794229e-09,0.0004877871539622,1.3389321073937e-08,0.0014633614691031
-3.6813238467382e-05,2.292018317801e-07,2.0730003371637e-08,0.00017649528628879,3.5938497133611e-07,6.0770859402987e-09,0.00049084317062766,1.5984358031291e-08,0.0014725295374668
-3.7053900390751e-05,2.4066192336911e-07,2.0838338159359e-08,0.00017559839837397,3.8725336198487e-07,6.7865062542689e-09,0.0004940520030915,1.8723026602911e-08,0.0014821560399455
-3.7306595410289e-05,2.5269501953756e-07,2.0950868947544e-08,0.0001746786083703,4.1665103781055e-07,7.5356601696638e-09,0.00049742127007757,2.1613728676081e-08,0.0014922638072822
-3.7571925180803e-05,2.6532977051444e-07,2.106770804234e-08,0.00017373615452135,4.4766640662469e-07,8.326906460414e-09,0.00050095899614752,2.4665384302904e-08,0.001502876970245
+time,dt,elastic_strain_xx,elastic_strain_yy,matl_ts_min,max_disp_x,max_disp_y,max_hydro,plasticity_strain_xx,plasticity_strain_yy,plasticity_strain_zz,sigma_xx,sigma_yy
+0,0,0,0,0,0,0,0,0,0,0,0,0
+1e-05,1e-05,5.7142859496491e-09,-1.4285712602656e-09,1.7976931348623e+308,5.59352864834e-08,5.6333938186502e-10,0.00013333335277204,0,0,0,0.00040000002919565,2.5440424343982e-11
+2.05e-05,1.05e-05,1.171428590898e-08,-2.9285712722289e-09,1.7976931348623e+308,1.1466733615868e-07,1.1548458299262e-09,0.00027333335079378,0,0,0,0.00082000002510949,2.2961799847035e-11
+3.1525e-05,1.1025e-05,1.8014285672933e-08,-4.5035714013047e-09,1.7976931348623e+308,1.7633598936055e-07,1.7759276036318e-09,0.00042033335104587,0,0,0,0.0012609999980533,1.8960035635691e-12
```

Comment:
Differences are pretty small wrt. previous gold file, right? Probably due to solver tolerances...

Reply:
Yes, the effect of this change is negligible (it wasn't diffing at all). I had to re-gold because I added a few more output variables.